core/templates/dev/head/pages/signup-page/signup-page.directive.html

```diff
@@ -84,3 +84,8 @@ <h2 class="oppia-signup-page-title" translate="I18N_SIGNUP_COMPLETE_REGISTRATION
     </form>
   </md-card>
 </div>
+<style>
+  .oppia-long-text > label {
```

Comment:
Is .oppia-long-text usd in other files? If so this needs to be placed in a more general location like oppia.css.

Reply:
it will be scoped through this file only, now I've added super-class now its specific to the element Thanks!